src/main/java/frc/robot/commands/IntakeSequence.java

```diff
@@ -1,46 +1,62 @@
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Cal;
 import frc.robot.subsystems.Intake;
 import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.Lift.LiftPosition;
 
-public class IntakeSequence extends CommandBase {
+public class IntakeSequence extends SequentialCommandGroup {
   private Intake intake;
   private Lift lift;
 
   public IntakeSequence(Intake intake, Lift lift) {
     this.intake = intake;
     this.lift = lift;
```

Comment:
Add required subsystems (can you do this for outtake too, they merged already)

Reply:
![image](https://user-images.githubusercontent.com/22648713/217885029-936f8f34-5475-468f-a9ec-7946363c321a.png)
Seems like OuttakeSequence has it already